packages/headless-ui/vaul/test/src/app/controlled/page.tsx

```
@@ -0,0 +1,206 @@
+              <div className="mx-auto w-12 h-1.5 flex-shrink-0 rounded-full bg-zinc-300 mb-8" />
+              <div className="max-w-md mx-auto">
+                <Drawer.Title className="font-medium mb-4">Unstyled drawer for React.</Drawer.Title>
+                <p className="text-zinc-600 mb-2">
+                  This component can be used as a replacement for a Dialog on mobile and tablet devices.
+                </p>
+                <p className="text-zinc-600 mb-8">
+                  It uses{' '}
+                  <a
+                    href="https://www.radix-ui.com/docs/primitives/components/dialog"
+                    className="underline"
+                    target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)